app/src/services/votingService.ts

```diff
@@ -65,4 +81,29 @@ export class VotingService implements IVotingService {
 
     return votingContract
   }
+
+  async getContract(votingAddress: string): Promise<Contract> {
+    const contractService = this.getContractService(votingAddress)
+
+    return await contractService.getContract()
+  }
+  private getContractService(votingAddress: string): SmartContract {
+    return new SmartContract(votingAddress, VOTING_ABI)
+  }
+  private async deployVotingContract(): Promise<string> {
+    const votingImplementation = await this.getContract(VOTING_IMPL_ADDRESS)
+    console.log('Voting implementation:', votingImplementation.interface.fragments)
+    const votingProxyFactory = await this.web3Library.getFactoryContract(
+      BEACON_PROXY_ABI,
+      BEACON_PROXY_BYTECODE
+    )
+    const beaconProxyDeployment = await votingProxyFactory.deploy(
+      VOTING_BEACON_ADDRESS,
+      votingImplementation.interface.encodeFunctionData('initialize', [])
```

Comment:
this can be '0x' since we dont have constructor function

Reply:
As in
```typescript
      votingImplementation.interface.encodeFunctionData('initialize', ['0x'])
```
?

Reply:
just plain '0x'